src/cli/cli_history.cpp

```diff
@@ -1614,6 +1614,76 @@ template <> otError History::Process<Cmd("omrprefix")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli history onlinkprefix
+ * @code
+ * history onlinkprefix
+ * | Age                  | On-link Prefix                                   |IsLocal |
+ * +----------------------+--------------------------------------------------+--------+
+ * |         00:00:50.600 | 2001:efc6:75a8:efee::/64                         | no     |
+ * |         00:11:04.327 | fd74:fe69:9f21:437::/64                          | yes    |
+ * Done
+ * @endcode
+ * @code
+ * history onlinkprefix list
+ * 00:00:50.600 -> on-link-prefix:2001:efc6:75a8:efee::/64 is-local:no
+ * 00:11:04.327 -> on-link-prefix:fd74:fe69:9f21:437::/64 is-local:yes
+ * Done
+ * @endcode
+ * @cparam history onlinkprefix [@ca{list}] [@ca{num-entries}]
+ * * Use the `list` option to display the output in list format. Otherwise, the output is shown in table format.
+ * * Use the `num-entries` option to limit the output to the number of most-recent entries specified. If this option
+ *   is not used, all stored entries are shown in the output.
+ * @par
+ * Displays the favored on-link prefix history in table or list format.
+ * @par
+ * Each table or list entry provides:
+ * * Age: Time elapsed since the command was issued, and given in the format:
+ *        `hours`:`minutes`:`seconds`:`milliseconds`
```

Comment:
Should it be a dot between seconds and milliseconds?

`hours`:`minutes`:`seconds`.`milliseconds`